tests/Unit/NumericalAlgorithms/Lapack/Test_GeneralizedEigenvalue.cpp

```diff
@@ -0,0 +1,67 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include <cstddef>
+
+#include "DataStructures/DataVector.hpp"
+#include "DataStructures/Matrix.hpp"
+#include "NumericalAlgorithms/Lapack/GeneralizedEigenvalue.hpp"
+#include "tests/Unit/TestingFramework.hpp"
+
+namespace {
+void test_generalized_eigenvalue() {
+  // Set up some test matrices
+  Matrix A(2, 2, 0.0);
+  Matrix B(2, 2, 0.0);
+  A(0, 0) = 1.0;
+  A(0, 1) = 2.0;
+  A(1, 0) = -3.0;
+  A(1, 1) = -4.0;
+  B(0, 0) = 4.0;
+  B(0, 1) = -3.0;
+  B(1, 0) = -2.0;
+  B(1, 1) = 1.0;
```

Comment:
With one of the upcoming PRs you can rewrite this as `Matrix A{{1., 2.}, {-3., -4.}}`. Don't know if we want to get that in first?

Reply:
I would say it's not worth holding up merging into master. Whichever PR is done first, the other will fix these lines when rebasing onto develop.